lib/iris/tests/integration/test_trajectory.py

```diff
@@ -127,43 +149,61 @@ def traj_to_sample_points(trajectory):
                              287.43795776, 287.59701538, 287.52468872,
                              287.45025635, 287.52716064], dtype=np.float32)
 
-        self.assertCML(trajectory_cube, ('trajectory', 'zigzag.cml'), checksum=False)
+        self.assertCML(trajectory_cube, ('trajectory', 'zigzag.cml'),
+                       checksum=False)
         self.assertArrayAllClose(trajectory_cube.data, expected, rtol=2.0e-7)
 
-    @tests.skip_data
-    def test_tri_polar(self):
+
+@tests.skip_data
+class TestTriPolar(tests.IrisTest):
+    def setUp(self):
         # load data
-        cubes = iris.load(tests.get_data_path(['NetCDF', 'ORCA2', 'votemper.nc']))
+        cubes = iris.load(tests.get_data_path(['NetCDF', 'ORCA2',
+                                               'votemper.nc']))
         cube = cubes[0]
         # The netCDF file has different data types for the points and
         # bounds of 'depth'. This wasn't previously supported, so we
         # emulate that old behaviour.
-        cube.coord('depth').bounds = cube.coord('depth').bounds.astype(np.float32)
-
-        # define a latitude trajectory (put coords in a different order to the cube, just to be awkward)
+        b32 = cube.coord('depth').bounds.astype(np.float32)
+        cube.coord('depth').bounds = b32
+        self.cube = cube
+        # define a latitude trajectory (put coords in a different order
+        # to the cube, just to be awkward)
         latitudes = list(range(-90, 90, 2))
         longitudes = [-90]*len(latitudes)
-        sample_points = [('longitude', longitudes), ('latitude', latitudes)]
+        self.sample_points = [('longitude', longitudes),
+                              ('latitude', latitudes)]
+
+    def test_tri_polar(self):
 
         # extract
-        sampled_cube = iris.analysis.trajectory.interpolate(cube, sample_points)
-        self.assertCML(sampled_cube, ('trajectory', 'tri_polar_latitude_slice.cml'))
+        sampled_cube = iris.analysis.trajectory.interpolate(self.cube,
+                                                            self.sample_points)
+        self.assertCML(sampled_cube, ('trajectory',
+                                      'tri_polar_latitude_slice.cml'))
 
+    def test_tri_polar_exceptions(self):
         # Try to request linear interpolation.
         # Not allowed, as we have multi-dimensional coords.
-        self.assertRaises(iris.exceptions.CoordinateMultiDimError, iris.analysis.trajectory.interpolate, cube, sample_points, method="linear")
+        self.assertRaises(iris.exceptions.CoordinateMultiDimError,
+                          iris.analysis.trajectory.interpolate, self.cube,
+                          self.sample_points, method="linear")
 
         # Try to request unknown interpolation.
-        self.assertRaises(ValueError, iris.analysis.trajectory.interpolate, cube, sample_points, method="linekar")
+        self.assertRaises(ValueError, iris.analysis.trajectory.interpolate,
```

Comment:
As you're tidying up here, you could also split these two error modes into two separate test cases.